src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetPackageSearchService.cs

```diff
@@ -196,6 +196,56 @@ public NuGetPackageSearchService(ServiceActivationOptions options, IServiceBroke
             return await GetPackageVersionsAsync(identity, packageSources, includePrerelease, isTransitive, projects: null, cancellationToken);
         }
 
+        public async Task<string> GetPackageReadMeAsync(PackageIdentity identity, IReadOnlyCollection<PackageSourceContextInfo> packageSources, bool includePrerelease, CancellationToken cancellationToken)
```

Comment:
`GetPackageReadMeAsync` and `GetPackageHasReadMeAsync` methods have almost the same code. I am thinking if the `TryGetPackageReadMeAsync` pattern would be a better fit here because if a package has a readme, the method returns true along with the readme content; otherwise, it returns false.

Reply:
I like it, done.